### ISSUE TITLE
A11y - Text does not have enough color contrast

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1363]
## User story

Text should have color contrast following AA WCAG guidelines: https://webaim.org/standards/wcag/checklist#sc1.4.3

## Acceptance criteria

Text and images of text have a contrast ratio of at least 4.5:1.
Large text - at least 18 point (typically 24px) or 14 point (typically 18.66px) and bold - has a contrast ratio of at least 3:1.

## Additional info

there are multiple ways to achieve this:

- increase font size site-wide to 18px. Probably a good idea since font size is really small, and size 18px looks pretty good and more easily readable. The con is that it takes significantly more space and might distort layout, so it might be time-consuming or impractical in some case
- increase font size site-wide to 14px and make certain text elements bold (links, text on gray, buttons...) to satisfy contrast criterion of 3/1
- leave fonts as is and change the color of specific text elements to fulfill contrast threshold of 4.5/1

An exhaustive list of cases is extensive to produce, because it applies to virtually all pages on the site, so a better approach is to identify specific use cases and create design tokens (classes) for each that are a11y out of the box and constrain its usage

Examples of low contrast elements:

- white text on primary color bg (primary buttons, login links in header, ...)
- text over images, namely hero banner
- any colored text
  - green links over any background
  - dataset type count in homepage
  - any muted / gray text, e.g. tab headers in article detail, beadcrumbs
  - red error text in https://beta.gigadb.org/user/create
- text over gray background, e.g. KPIs in home, team cards, article subtitle in article detail

**Comment by luistoptal** [comment]
A good idea to finish and merge this first, as there is some work related to color contrast: https://github.com/gigascience/gigadb-website/issues/1374
